File: rulr/Nodes/Item/Camera.py
import rulr.Nodes
import rulr.Components.RigidBody
import rulr.Components.View

class Node(rulr.Nodes.Base):
	def __init__(self):
		super().__init__()
		self.components.rigidBody = rulr.Components.RigidBody.Component()
		self.components.view = rulr.Components.View.Component(self.components.rigidBody)

File: rulr-web/src/rulr-web/Nodes/Item/Camera.js
//Item.Camera
import { Base } from '../Base.js'
import * as Debug from '../../Utils/Debug.js'

class Node extends Base {
	constructor() {
		super();
	}

	async init() {
		await super.init();

		this.viewportCamera = new THREE.PerspectiveCamera();
		this.viewportCameraPreview = new THREE.CameraHelper(this.viewportCamera);
		
		this.viewportObject.add(this.viewportCameraPreview);
	}
}
Debug.wrapClassPrototypeMethods(Node);

export { Node };